### PR TITLE
gpu doc refresh

### LINKED ISSUE #593: need a section for azure gpu workers
We discuss GPU workers on AWS and GCP here:

https://ubuntu.com/kubernetes/docs/gpu-workers

But not Azure. We should figure out what instance-types are necessary to document how to add GPU workers on Azure.

### PR DESCRIPTION
**Pull request by kwmonroe**
- adjust flow for a less headache-inducing experience
- add azure instance types (aws and azure are similar; gcp is the odd-cloud out)
- bump `nvidia-test.yaml` image to current supported version for k8s 1.27
- update example syntax and output
- update links

Fixes #593

### CODE REVIEW

File: pages/k8s/gpu-workers.md [hunk 1]
```diff
@@ -13,96 +13,101 @@ layout: [base, ubuntu-com]
 toc: False
 ---
 
-**Charmed Kubernetes** supports GPU-enabled
-instances for applications which can use them. The kubernetes-worker charm will
-automatically detect NVIDIA hardware and enable the appropriate support.
-However, the implementation of GPU-enabled instances differs greatly between
-public clouds. This page outlines the recommended methods for running GPU
-enabled hardware for different public clouds.
+**Charmed Kubernetes** supports GPU-enabled instances for applications that
```
Comment (evilnick) [inline pages/k8s/gpu-workers.md:16]: in a strictly grammatical sense, either 'which' or 'that' are correct in this context, however we tend to prefer which for restrictive clauses as it also applies to non-restrictive clauses as well and can only really be misused when applied to people.

File: pages/k8s/gpu-workers.md [hunk 2]
```diff
@@ -159,35 +163,35 @@ You can inspect the logs to find the hardware report.
...
+|  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
+|        ID   ID                                                             Usage      |
+|=======================================================================================|
+|  No running processes found                                                           |
++---------------------------------------------------------------------------------------+
 ```
 
 <!-- LINKS -->
-[asset-nvidia]: https://raw.githubusercontent.com/juju-solutions/kubernetes-docs/main/assets/nvidia-test.yaml
+[asset-nvidia]: https://raw.githubusercontent.com/charmed-kubernetes/kubernetes-docs/main/assets/nvidia-test.yaml
```
Comment (evilnick) [inline pages/k8s/gpu-workers.md:189]: should we more properly add this asset to the bundle?

**Review by evilnick — APPROVED** [review]
Just a query over the location of the asset yaml - most of these were migrated to the bundle repo.
Thanks for the improvements :cake:

File: pages/k8s/gpu-workers.md (hunk 1 above)
Comment (kwmonroe) [reply pages/k8s/gpu-workers.md:16]: :broom:

File: pages/k8s/gpu-workers.md (hunk 2 above)
Comment (kwmonroe) [reply pages/k8s/gpu-workers.md:189]: Let's do 'em all in #765

**Comment by kwmonroe** [comment]
> Just a query over the location of the asset yaml

Let's keep this asset here for now with the expectation that we'll re-home all these in one fell swoop with #765.

**Pull request merged**